rust-toolchain.toml

```diff
@@ -1,9 +1,3 @@
-# If you see this, run "rustup self update" to get rustup 1.23 or newer.
-
-# NOTE: above comment is for older `rustup` (before TOML support was added),
-# which will treat the first line as the toolchain name, and therefore show it
-# to the user in the error, instead of "error: invalid channel name '[toolchain]'".
-
 [toolchain]
 channel = "1.76"                     # Needed for deno & cts_runner. Firefox's MSRV is 1.74
```

Comment:
thought(non-blocking): @cwfitzgerald, I feel like this comment might not be carrying its weight, considering that it's outdated (Firefox 127 has updated to 1.76 already), and it's hard to detect when it needs updating. Any reason we shouldn't let MSRVs in CI and Cargo manifests speak for themselves?